src/pages/components/CertificateTimeInterval.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react'
+import styled from 'styled-components'
+
+import StartEndSpacerSVG from '../../assets/images/start-end-spacer-bg.svg'
+
+export default function CertificateTimeInterval({ sTimestamp, eTimestamp }) {
+  function calcTimeInterval(s, e) {
+    const i = new Date(e) - new Date(s)
+    return i
+  }
+  function convIntervalToHoursAndMins(i) {
+    const m = Math.floor(i / (1000 * 60)) % 60
+    const h = Math.floor(i / (1000 * 60 * 60))
+    const str = `${h.toString().padStart(2, '0')}H ${m.toString().padStart(2, '0')}M`
+    return str
+  }
+  function convIntervalToMonthsAndDays(i) {
+    let d = Math.floor(i / (1000 * 60 * 60 * 24)) % 30
+    let m = Math.floor(i / (1000 * 60 * 60 * 24 * 30))
+    let str = `${m.toString().padStart(2, '0')}m ${d.toString().padStart(2, '0')}d`
+    if (m > 99) {
+      d = 0
+      m = 99
+      str = `>${m.toString().padStart(2, '0')}m${d.toString().padStart(2, '0')}d`
+    }
+    return str
+  }
+  function convIntervalToStr(s, e) {
+    const i = calcTimeInterval(s, e)
+    if (i > 1000 * 24 * 60 * 60) {
+      return convIntervalToMonthsAndDays(i)
+    } else {
+      return convIntervalToHoursAndMins(i)
+    }
+  }
```

Comment:
useCallback

Reply:
Can I just have only one function which is `convIntervalToStr` would that work better ?

Reply:
Actually I will move this outside the `export default function`